docs/resources/Channel.md

```diff
@@ -302,6 +321,18 @@ Post a message to a guild text or DM channel. If operating on a guild channel, t
 | nonce | snowflake | a nonce that can be used for optimistic message sending | false |
 | tts | bool | true if this is a TTS message | false |
 
+## Create Reaction % PUT /channels/{channel.id#DOCS_CHANNEL/channel-objects}/messages/{message.id#DOCS_CHANNEL/message-object}/reactions/{emoji}/@me
+
+Create a reaction for the message. If nobody else has reacted to the message using this emoji, this endpoint requires the 'ADD\_REACTIONS' permission to be present on the current user. Returns a 204 empty response on success.
+
+## Delete Own Reaction % DELETE /channels/{channel.id#DOCS_CHANNEL/channel-objects}/messages/{message.id#DOCS_CHANNEL/message-object}/reactions/{emoji}/@me
```

Comment:
you are missing the GET endpoint for listing all users that reacted with the emoji